code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -490,4 +490,8 @@
 
 /mob/living/simple_animal/Destroy()
 	GLOB.simple_mob_list.Remove(src)
-	..()
\ No newline at end of file
+	..()
+
+/mob/living/simple_animal/restrained()
+	if(buckled && istype(buckled, /obj/effect/energy_net/safari))
```

Comment:
Does this really need to be constrained to a subtype?

Reply:
I wasn't sure if anything else buckles `simple_animal`s that warranted removing their ability to attack. ie, buckling a panther to a chair imo should only count as `PARTIALLY_BUCKLED`, rather than `FULLY_BUCKLED`. I believe it's the same for humans